src/main/java/com/radixdlt/crypto/SHAHashHandler.java

```diff
@@ -28,22 +31,32 @@
 	private final ThreadLocal<MessageDigest> hash512Digester = ThreadLocal.withInitial(() -> getDigester("SHA-512"));
 
 	SHAHashHandler() {
+		// cannot construct
```

Comment:
Should throw IllegalStateException instead of just a comment

Reply:
Ah my bad, this constructor is in fact used.